sagemaker/launch/launch_utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+
+import os
+import shutil
+import tarfile
+
+
+def wrap_model_artifacts(path_to_model, path_to_yaml, path_to_json, path_to_entry,
+                         output_path, output_tarfile_name='model'):
+    """ A utility function to zip model artifacts into a tar package
+
+    According to SageMaker's specification of the `Model Directory Structure
+    https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/using_pytorch.html#model-directory-structure`_,
+    this function will put the entry point file into a sub-folder,named `code`, and then zip the
+    `model.bin`, the `**.yaml`, the `**.json` file, and the `code` sub-folder into a tar package
+    with the name specified by the wrap_name argument, and save it to the `output_path`.
+
+    This function will check if given files exist. Non-existance will raise an error.
+    
+    Parameters
+    ----------
+    path_to_model: str
+        The path of GraphStorm's `model.bin` file.
+    path_to_yaml: str
+        The path of the YAML file generated by GraphStorm during model training.
+    path_to_json: str
+        The path of JSON file generated by GraphStorm gconstruct or GSProcessing.
+    path_to_entry: str
+        The path of the entry point file for a specific task. The file will be put into a
+        sub-folder, named 'code'.
+    output_path: str
+        The folder where the output tar package will be saved. If not provided, will
+        raise an error.
+    output_tarfile_name: str
+        The name of the tar package. Default is `model`.
+
+    """
+    # check if files exist
+    assert os.path.exists(path_to_model), f'The model file, {path_to_model}, does not exist.'
+    assert os.path.isfile(path_to_model), f'The model file, {path_to_model}, should be a file \
+                                            path, but got a folder.'
+    assert os.path.exists(path_to_yaml), f'The model configuration YAML file, {path_to_yaml}, \
+                                           does not exist.'
+    assert os.path.isfile(path_to_yaml), f'The model configuration YAML file, {path_to_yaml}, \
+                                           should be a file path, but got a folder.'
+    assert os.path.exists(path_to_json), f'The graph metadata JSON file, {path_to_json}, does \
+                                           not exist.'
+    assert os.path.isfile(path_to_json), f'The graph metadata JSON file, {path_to_json}, should \
+                                           be a file path, but got a folder.'
+    assert os.path.exists(path_to_entry), f'The SageMaker entry point file, {path_to_entry}, does \
+                                           not exist.'
+    assert os.path.isfile(path_to_entry), f'The SageMaker entry point file, {path_to_entry}, \
+                                           should be a file path, but got a folder.'
+
+    # create the output folder if not exist.
+    assert os.path.isdir(output_path), 'Output path should be a folder name, but got a ' + \
```

Comment:
What will happen if output_path does not exist?

Reply:
Add one test case.